magento/magento2
# Fixed wrong Button style on wishlist page in buttons #19564

### Description (*)


### Fixed Issues (if relevant)

1. magento/magento2#19564: Wrong Button style on wishlist page in buttons

### Manual testing scenarios (*)

1. Login in your frontend.
2. Add any product to Wishlist.
3. Go to Wishlist page
4. You can see Grey button

### Expected result

1. Buttons should be Blue background color.
![expected](https://user-images.githubusercontent.com/23443991/49495449-01d8e600-f889-11e8-964a-eb677fdb5c71.jpg)

### Actual result

1. Buttons background color was Grey.
![actual](https://user-images.githubusercontent.com/23443991/49495446-000f2280-f889-11e8-9ddb-bb992af8ab52.jpg)

### Contribution checklist (*)
